flask/src/main/java/net/corda/gradle/flask/FlaskJarTask.java

```diff
@@ -218,12 +225,14 @@ public WorkResult execute(@Nonnull CopyActionProcessingStream copyActionProcessi
                  */
                 File temporaryJar = File.createTempFile("premature", ".zip", getTemporaryDir());
                 try (ZipOutputStream zipOutputStream = new ZipOutputStream(Flask.write(temporaryJar, true))) {
+                    zipOutputStream.setLevel(BEST_COMPRESSION);
```

Comment:
we surely don't want to use `BEST_COMPRESSION` (that takes a lot of cpu resources) for a temporary jar we are going to immediately discard

Reply:
True; I can switch that to `NO_COMPRESSION`.